model/project_aliases.go

```diff
@@ -185,26 +185,18 @@ func findMatchingAliasForRepo(repoID, alias string) ([]ProjectAlias, error) {
 // Typically FindAliasInProjectRepoOrConfig should be used.
 // Returns true if we have an alias match or the alias doesn't match but
```

Comment:
Remove this part of the doc comment since it's no longer relevant.

Reply:
done